src/frontend/src/catalog/system_catalog/rw_catalog/rw_secrets.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_common::types::Fields;
+use risingwave_frontend_macro::system_catalog;
+
+use crate::catalog::system_catalog::SysCatalogReaderImpl;
+use crate::error::Result;
+
+#[derive(Fields)]
+struct RwSecret {
+    #[primary_key]
+    id: i32,
+    name: String,
+    owner: i32,
+    acl: String,
```

Comment:
wish we could add a `backend` column for "meta" or "HashiCrop_Vault".

Reply:
Not viable for now because it's encrypted. We figure out what we can do after we add hashicorp